src/components/Incorporation/UploadRules.vue

```diff
@@ -74,9 +74,35 @@
             </li>
           </ul>
 
+          <p class="mt-4">
+            The Cooperative Association Regulation provide a
+            <a href="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/391_2000#ScheduleB">Schedule B</a>
+            Schedule B template of Rules. The rules adopted by a Cooperative Association may be in this form, a modified
+            form of these Rules or in another form altogether.
+          </p>
+
           <!-- help section 3 -->
-          <p id="help-text-section-3"
-            v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section3.items"
+          <p class="help-section-title font-weight-bold mt-4">
+            {{getCreateRulesResource.helpSection.helpText.section3.label}}
+          </p>
+          <ul class="bulleted-list mt-4">
+            <li
+              v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section3.items"
+              class="mt-2"
+              :key="index"
+              v-html="item"
+            />
+          </ul>
+
+          <p class="mt-4">
+            Before choosing to use the
+            <a href="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/391_2000#ScheduleB">Schedule B</a>
```

Comment:
Sure.

Reply:
Looks good.

Reply:
I put the url to under "getCreateRulesResource.helpSection.helpText.section3.href". I think it's more consistent this way since it is under section3. What do you think?

Reply:
I agree with that. On the other hand, the paragraph in which this link is used is itself hard-coded in the component. I don't care -- this is a Coop-only component anyway atm.

Reply:
Yes. I'm aware of that too. This UploadRules component is only used for Coop, so I thought hard-coding these two paragraphs, for the schedule B url, is most-likely fine. I would put it to the resource if this component was shared among others.